docs/swarm/swarm-mode.md

```diff
@@ -0,0 +1,176 @@
+test a services based upon images you've created or other available images. In
+your production environment, swarm mode provides a fault-tolerant platform with
+cluster management features to keep your services running and available.
+
+If you haven't already, read through the [swarm mode key concepts](key-concepts.md)
+and try the [swarm mode tutorial](swarm-tutorial/index.md).
+
+## Create a cluster
+
+When you run the command to create a swarm, the Docker Engine starts running in
+a swarm mode. Run [`docker swarm init`](/engine/reference/commandline/swarm_init.md)]
+to create a single-node swarm on the current node.
+
+The `docker swarm init` command does the following:
+
+* switches the Docker Engine on the current node into swarm mode.
+
+* creates a swarm named `default`.
+
+* designates the current node as a manager node for the swarm.
+
+* configures the manager to listen on an active network interface on port 2377.
+
+    >**Note**: Docker recommends you use the `--listen-addr` flag to
+    specify precisely the IP address and port where the manager should
+    listen.
+
+* Creates a root CA for the swarm and implements the default security policy.
+   More in depth documentation about about mutual TLS Authentication is forthcoming.
+
+* Implements a raft consensus model for managing the swarm state and the desired
+state for services.
+
+More topics are forthcoming to help you plan for deploying services into
+a production swarm.
+
+### Configure node listen address
+
+When you initialize a swarm the Engine automatically configures a listen address
+for manager nodes based upon the available network interfaces.
+
+The manager node uses the listens address for cluster management
+communications, such as requests for new nodes to join the swarm. The
+other nodes on the swarm must be able to access the manager node on the
+IP address and port you specify for the listen address.
+
+Especially when there are multiple active network interfaces, Docker
+recommends you explicitly define the listen address when you initialize
+a swarm:
+
+  ```
+  docker swarm init --listen-addr <MANAGER-IP>:<PORT>
+  ```
+
+### Set acceptance policy
+
+When a new node attempts to join the swarm, the swarm manager enforces
+an acceptance policy. You can configure the swarm to automatically
+accept new nodes into the swarm as follows:
+
+* **Do not automatically accept nodes**: From a manager node, you must approve
+requests to join the swarm from all manager nodes and all worker nodes.
+
+    ```
+    docker swarm init --auto-accept none
+    ```
+
+    This is the most secure option. However, it requires intervention to
+    accept every new node into the swarm.
+
+* **Automatically accept workers**: The swarm manager automatically
+approves any requests to join the swarm from worker nodes. From a
+manager node, you must approve all manager nodes requesting to join the
+swarm.
+
+    ```
+    docker swarm init --auto-accept worker
+    ```
+
```

Comment:
Yes, it could be totally fine in production, e.g., if you are running all machines on an isolated network and aren't concerned about rogue nodes joining the swarm.  It's up to them; I don't think we should be prescriptive here.